google-cloud-clients/google-cloud-spanner/src/main/java/com/google/cloud/spanner/InstanceAdminClientImpl.java

```diff
@@ -0,0 +1,213 @@
+/*
```

Comment:
are these chances part of the overall refactoring?

Reply:
Yes, this class used to be an inner class of SpannerImpl.